Directory.Build.props

```diff
@@ -66,4 +66,12 @@
     <DebugType Condition="'$(DebugType)' == ''">embedded</DebugType>
     <MonoBuild>true</MonoBuild>
   </PropertyGroup>
+  <!-- Configuration MSBuild for portable (xcopy-install) toolsets: works on WinNT and linux/mac via Mono, isolates from machine environment:
+    uses only tools installed with it, ignores Registry and GAC and Visual Studio installations to provide the same build experience on all machines -->
+  <PropertyGroup Condition="'$(Configuration)' == 'MachineIndependent'">
```

Comment:
Rather than making the configuration space more complicated, what if you introduce a simple property called `MachineIndependent`? That way you can have Release / Debug machine independent.

Reply:
I'm not sure about it because separate configuration allows edit project files in IDE with correctly #ifdef states. An example I found problems in compilation some registry dependent tests when I disable registry.

Reply:
Good point, didn't think about the IDE scenario :). New config it is then.